### ISSUE TITLE
 Failed: No such file or directory - /var/vcap/jobs/powerdns/bin/powerdns_ctl 

### ISSUE THREAD

**Issue by chrisrana** [cloudfoundry/bosh-init#109]
I am using bosh-init,bosh-257.9.tgz and cf-233 .Stemcell 3262.2.

I get this error when i comment out powerdns from bosh-init.yml file.
 - {name: powerdns, release: bosh}

`<%
      load 'bosh-init/bosh-init.rb'
%>

name: <%= $os_tenant %>

releases:
- name: bosh
  url: '<%= $bosh_director_release_url %>'
  sha1: <%= $bosh_director_sha %>
- name: bosh-openstack-cpi
  url: <%= $bosh_openstack_cpi %>
  sha1: <%= $bosh_openstack_cpi_sha %>

resource_pools:
- name: vms
  network: private
  stemcell:
    url: <%= $stemcell_path %>
    sha1: <%= $stemcell_sha1 %>
  cloud_properties:
    instance_type: <%= $bosh_flavor %>
    #availability_zone: soltes2

disk_pools:
- name: disks
  disk_size: 20_000

networks:
- name: private
  type: manual
  subnets:
  - range: <%= $cidr %> # <--- Replace with a private subnet CIDR
    gateway: <%= $gateway_ip %> # <--- Replace with a private subnet's gateway
    dns: [10.20.0.3] # <--- Replace with your DNS
    cloud_properties: {net_id: <%= $net_id %>} # <--- # Replace with private network UUID
- name: public
  type: vip

jobs:
- name: bosh
  instances: 1

  templates:
  - {name: powerdns, release: bosh}
  - {name: nats, release: bosh}
  - {name: postgres, release: bosh}
  - {name: blobstore, release: bosh}
  - {name: director, release: bosh}
  - {name: health_monitor, release: bosh}
  - {name: registry, release: bosh}
  - {name: openstack_cpi, release: bosh-openstack-cpi}

  resource_pool: vms
  persistent_disk_pool: disks

  networks:
  - name: private
    static_ips: [<%= $bosh_director_ip %>] # <--- Replace with a private IP
    default: [dns, gateway]

  properties:
    nats:
      address: 127.0.0.1
      user: nats
      password: nats-password

    postgres: &db
      listen_address: 127.0.0.1
      host: 127.0.0.1
      user: postgres
      password: postgres-password
      database: bosh
      adapter: postgres

    registry:
      address: <%= $bosh_director_ip %> # <--- Replace with a private IP
      host: <%= $bosh_director_ip %> # <--- Replace with a private IP
      db: *db
      http: {user: admin, password: admin, port: 25777}
      username: admin
      password: admin
      port: 25777

    blobstore:
      address: <%= $bosh_director_ip %> # <--- Replace with a private IP
      port: 25250
      provider: dav
      director: {user: director, password: director-password}
      agent: {user: agent, password: agent-password}

    director:
      address: 127.0.0.1
      name: my-bosh
      db: *db
      cpi_job: openstack_cpi
      max_threads: 3
      user_management:
        provider: local
        local:
          users:
          - {name: <%= $bosh_username %>, password: <%= $bosh_password %>}
          - {name: hm, password: hm-password}

    hm:
      director_account: {user: hm, password: hm-password}
      resurrector_enabled: true

    dns:
      address: "127.0.0.1"
      domain_name: "microbosh"
      db: *db

    openstack: &openstack
      auth_url: "<%= $os_auth_url%>" # <--- Replace with OpenStack Identity API endpoint
      project: iwan-soltest # <--- Replace with OpenStack project name
      domain: VMS # <--- Replace with OpenStack domain name
      username: "<%= $os_username %>" # <--- Replace with OpenStack username
      api_key: "<%= $os_password %>" # <--- Replace with OpenStack password
      tenant:  "<%= $os_tenant%>"
      default_key_name: <%= $key_name %>
      default_security_groups: [vms-security-group]
      ignore_server_availability_zone: true

    agent: {mbus: "nats://nats:nats-password@<%=$bosh_director_ip %>:4222"} # <--- Replace with a private IP

    ntp: &ntp [0.pool.ntp.org, 1.pool.ntp.org]

cloud_provider:
  template: {name: openstack_cpi, release: bosh-openstack-cpi}

  ssh_tunnel:
    host: <%=$bosh_director_ip %> # <--- Replace with a floating IP
    port: 22
    user: vcap
    private_key: <%= $key_file %> # Path relative to this manifest file

  mbus: "https://mbus:mbus-password@<%=$bosh_director_ip %>:6868" # <--- Replace with a floating IP

  properties:
    openstack: *openstack
    agent: {mbus: "https://mbus:mbus-password@0.0.0.0:6868"}
    blobstore: {provider: local, path: /var/vcap/micro_bosh/data/cache}
    ntp: *ntp
`

**Comment by dpb587-pivotal** [comment]
If you want to remove the `powerdns` job, you'll need to set `dns` to false (currently you have `address`, `domain_name`, and `db` set). Otherwise it appears to the director that it should be talking to dns during your deployments.

Also, for easier readability of snippets, please consider using code fences or a separate gist...

    ```
    name: <%= $os_tenant %>
    releases:
       - more stuff...
    ```

**Comment by chrisrana** [comment]
Thanks for quick response.I will take care snippets using gist in future.
by the way how to set it false 

`dns:  false `

is it ?

**Comment by dpb587-pivotal** [comment]
Yes; should be placed inside your `jobs[0].properties` section - by [default](http://bosh.io/jobs/director?source=github.com/cloudfoundry/bosh&version=257.9#p=dns) it's expecting dns to be used.
